pkg/minikube/cruntime/cri.go

```diff
@@ -331,16 +331,27 @@ plugin_dirs = [
 `
 )
 
+// getCrictlPath returns the absolute path of crictl
+func getCrictlPath(cr CommandRunner) string {
+	cmd := "crictl"
+	rr, err := cr.RunCmd(exec.Command("which", cmd))
```

Comment:
you could also use [exec.LookPath](https://golang.org/pkg/os/exec/#example_LookPath) here, which should be easier than parsing stdout

Reply:
Not really (I do this mix-up myself, all the time), since it runs locally rather than remotely

Reply:
oh of course!

Reply:
I actually think the docker runtime has that problem, but it is not involved in this particular PR